proto/felixbackend.proto

```diff
@@ -548,3 +551,7 @@ message WireguardEndpointRemove {
   // The name of the wireguard host.
   string hostname = 1;
 }
+
+message GlobalBGPConfigUpdate {
+  repeated string service_cluster_ips = 1;
```

Comment:
`service_cluster_cidrs`?

Reply:
(I had that in my mind too, but forgot to action it.)